site/src/pages/components/menu.explainer.mdx

```diff
@@ -144,8 +145,31 @@ We should support specific use cases of interactive elements outside the `<menui
 
 <img src="/images/menubar-search-input.png" alt="menubar examples with search input as a sibling of menuitems" />
 
+## Invoking a menulist from a button element
 
-## Questions
+While a `<menubar>` contains a collection of `<menuitem>` elements that can be used to invoke their own submenu (`<menulist>` elements), `<menubar>`s are commonly used for more global or multi-component-level purposes.  If an author needs to provide a `<menulist>` of commands to interact with specific content or components of an interface, then a `<button>` element is commonly used to invoke a `<menulist>`.
+
+For instance, `<button>` elements that are used to reveal menus are commonly used in tables/grids to allow for adjustments to the presentation of the tabular data, if not to invoke the editing or deleting of specific row or cell content.
+
+<img src="/images/menu-table-button.jpg" alt="a button invokes a menulist in a table column header, revealing the menuitems to resize, reorder or sort by the column of which it was invoked." />
+
+Toolbars or other groupings of related controls within complex web interfaces may need to include buttons that can perform _many_ actions - the specific action to perform is chosen by the user when they interact with the primary button, invoking its menu of commands.
+
+<img src="/images/menu-youtube-button.jpg" alt="the youtube player consists of multiple buttons that invoke their own menulists. The settings button invokes a menulist allowing for the adjustment of quality, playback speed, toggling annotations and more." />
+<img src="/images/menu-split-button.jpg" alt="the create new button in github's top bar invokes a menulist to add new or import a repository, create a new codepace or gist, or make a new orginization or project." />
+
+Leveraging the `command` and `commandfor` attributes, it is proposed that `<button>` elements, outside of the context of a `<menubar>`, can invoke `<menulist>` elements.  A `<button>` that invokes a `<menulist>` would have an implicit `aria-haspopup=menu` property, along with an impliict `aria-expanded` state, reflecting whether the element is rendered (expanded) or not (collapsed). 
+It would not have an implicit `aria-details` property. This is because unlike other more generic popovers, keyboard focus should immediately move from the invoking button to the first focusable element, or element with the autofocus attribute, of the rendered `<menulist>`.
```

Comment:
Thanks for spelling out this a11y guidance! It will help us implement this later.

Reply:
Yep, +1 this is huge.